src/Controls/src/Core/Platform/Android/ImageSourceLoader.cs

```diff
@@ -12,56 +12,10 @@ namespace Microsoft.Maui.Controls.Platform
 {
 	class ImageSourceLoader : IImageSourcePart
 	{
-		public IImageSource Source
```

Comment:
These all have just been moved into a shared class so all platforms can use this class